core/src/main/kotlin/org/evomaster/core/search/gene/GeneUtils.kt

```diff
@@ -13,6 +13,26 @@ object GeneUtils {
      */
     private val intpow2 = (0..30).map { 2.0.pow(it).toInt() }
 
+    enum class EscapeMode {
+        /**
+         * The [EscapeMode] enum is here to clarify the supported types of Escape modes.
+         *
+         * Different purposes require different modes of escape (e.g. URI may require percent encoding). This is to
+         * keep track of what modes are supported and how they map to the respective implementations.
+         *
+         * Any mode that is not supported will go under NONE, and will result in no escapes being applied at all. The
+         * purpose is to ensure that, even if the mode being used is unsupported, the system will not throw an exception.
+         * It may not behave as desired, but it should not crash.
+         *
+         */
+        URI,
```

Comment:
the `/** */` creates JavaDocs for the item following. I think such comment should be applied on the the enum `EscapeMode`, and not to its value `URI`